channeldb/graph.go

```diff
@@ -2284,8 +2284,12 @@ func (c *ChannelGraph) FilterChannelRange(startHeight,
 			rawCid := byteOrder.Uint64(k)
 			cid := lnwire.NewShortChanIDFromInt(rawCid)
 
+			// We need to initialize the time with 0 seconds,
+			// which equals `January 1, 1970, 00:00:00 UTC`
 			chanInfo := ChannelUpdateInfo{
-				ShortChannelID: cid,
+				ShortChannelID:       cid,
+				Node1UpdateTimestamp: time.Unix(0, 0),
+				Node2UpdateTimestamp: time.Unix(0, 0),
```

Comment:
👍  great catch

Reply:
perhaps we should add a `NewChannelUpdateInfo` constructor so that this is done by default & we dont need to remember to do it?